### DISCUSSION TITLE
Period in column name - How to query?

### DISCUSSION THREAD

**Discussion by pdutta777** [duckdb/duckdb#3242] [Q&A] [answered]
I have an arrow table which contains structs, and I am flattening those so the arrow fields look like "a.b" & "a.b.c". How do I run queries against this? I tried putting the column name in single quotes, but it just turns it into a string literal. Escaping the period with a backslash also does not work.

**Comment by Mytherin** [comment, marked as answer]
Double quotes can be used to query identifiers that contain special characters, e.g. `SELECT "a.b.c" FROM tbl`.
